migrations/00013_add_refresh_token_type.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
```

Comment:
does this work with hasura?

Reply:
it actually works, this must be new. They used to suggest using a different table for this. Nice!